internal/support/notifications/rest_subscription.go

```diff
@@ -22,29 +22,156 @@ import (
 
 	"github.com/edgexfoundry/edgex-go/internal/pkg"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
+	"github.com/edgexfoundry/edgex-go/internal/support/notifications/errors"
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 	"github.com/gorilla/mux"
 )
 
-const (
-	applicationJson = "application/json; charset=utf-8"
-)
+func getSubscriptions() (subscriptions []models.Subscription, err error) {
```

Comment:
One of the goals of this effort was to decouple this completely. Could you move these functions into their own file, so that the rest logic and the business logic don't live together?